server/parameters/CoreConfigParser.java

```diff
@@ -131,12 +148,12 @@ protected static class Stdout extends Compound<CoreConfig.Common.Output.Type.Std
                     );
                     private static final Predefined<Boolean> enable =
                             predefined("enable", "Enable logging to stdout.", BOOLEAN);
-                    private static final Set<Predefined<?>> parsers = set(typeParser, enable);
+                    private static final Set<Static> parsers = set(typeParser, enable);
```

Comment:
Static <- {Predefined, Optional} -- the names of the keys are predefined
Dynamic -- the names of the keys are NOT predefined